extensions/git/src/autofetch.ts

```diff
@@ -70,10 +71,18 @@ export class AutoFetcher {
 	private onConfiguration(): void {
 		const gitConfig = workspace.getConfiguration('git', Uri.file(this.repository.root));
 
-		if (gitConfig.get<boolean>('autofetch') === false) {
-			this.disable();
-		} else {
-			this.enable();
+		switch (gitConfig.get<string | boolean>('autofetch')) {
```

Comment:
Don't need to change this -- but FYI, you can change this to:
```ts
gitConfig.get<'all' | boolean>('autofetch')
```

to be more specific.